src/index.ts

```diff
@@ -18,14 +17,11 @@ const renderUsers = async (config: ApiConfig) => {
 };
 
 const app = () => {
-  const config = (window as any).__CONFIG__;
-  delete (window as any).__CONFIG__;
-
```

Comment:
You can register `'apiConfig'` right now, so there is no sense in moving this code to the ioc file.